docusaurus.config.ts

```diff
@@ -0,0 +1,189 @@
+  baseUrl: "/",
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: "fl0zone", // Usually your GitHub org/user name.
+  projectName: "fl0-docs", // Usually your repo name.
+
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+
+  markdown: {
+    mermaid: true,
+  },
+
+  themes: ["@docusaurus/theme-mermaid"],
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"],
+  },
+
+  presets: [
+    [
+      "classic",
+      {
+        docs: {
+          sidebarPath: "./sidebars.ts",
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl: "https://github.com/fl0zone/fl0-docs/tree/main/",
+        },
+        // blog: {
+        //   showReadingTime: true,
+        //   // Please change this to your repo.
+        //   // Remove this to remove the "edit this page" links.
+        //   editUrl:
+        //     "https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/",
+        // },
+        theme: {
+          customCss: "./src/css/custom.css",
+        },
+      } satisfies Preset.Options,
+    ],
+  ],
+
+  themeConfig: {
+    // Replace with your project's social card
+    image: "img/social-card.jpg",
+    colorMode: {
+      defaultMode: "dark",
+      disableSwitch: true,
+      respectPrefersColorScheme: false,
+    },
+    navbar: {
+      title: "FL0 Documentation",
+      logo: {
+        alt: "FL0 Logo",
+        src: "img/logo.svg",
+        className: "custom-navbar-logo-class",
+      },
+      items: [
+        {
+          to: "/docs/getting-started/crash-course",
+          label: "Getting Started",
+          position: "left",
+        },
+        {
+          type: "docSidebar",
+          sidebarId: "docsSidebar",
+          position: "left",
+          label: "Docs",
+        },
+        { to: "https://www.fl0.com/blog", label: "Blog", position: "left" },
+        {
+          href: "https://app.fl0.com",
+          label: "Platform",
+          position: "right",
+        },
+        {
+          href: "https://github.com/fl0zone/fl0-docs",
+          label: "GitHub",
+          position: "right",
+        },
+      ],
+    },
+    footer: {
+      style: "dark",
+      links: [
+        {
+          title: "Docs",
+          items: [
+            {
+              label: "Getting Started",
+              to: "/docs/getting-started/crash-course",
+            },
+          ],
+        },
+        {
+          title: "Community",
+          items: [
+            {
+              label: "LinkedIn",
+              href: "https://www.linkedin.com/company/fl0",
+            },
+            {
+              label: "Support - Discord",
```

Comment:
Consider renaming to "Support & Discussion (Discord)". If possible, could prefix with the Discord icon instead of using the (Discord) suffix.

Discord users will recognise the icons, while others won't know what a Discord is.

Reply:
Have made some improvements to the Discord links in #27